main.c

```diff
@@ -0,0 +1,23 @@
+
+
+#include "school.h"
+#include "dataBase.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#define _CRT_SECURE_NO_WARNINGS
+#define MAX_LINE_LENGTH 100
```

Comment:
Careful here since in dataBase.c you have the same define 
MAX_LINE_LENGTH but 150 and not 100